src/app/interface/navigation.interface.ts

```diff
@@ -0,0 +1,10 @@
+export interface NavigationParams {
```

Comment:
The name is too general. This is a set of navigation parameters specificly for dataset page.

Reply:
Changed NavigationParams ==> DatasetNavigationParams